routing/segment.hpp

```diff
@@ -0,0 +1,88 @@
+#pragma once
+
+#include "routing/road_point.hpp"
+
+#include "std/cstdint.hpp"
+#include "std/sstream.hpp"
+#include "std/string.hpp"
+
+namespace routing
+{
+// This is directed road segment used as vertex in index graph.
+//
+// You can imagine the segment as a material arrow.
+// Head of each arrow connected to fletchings of next arrows with invisible links:
+// these are the edges of the graph.
+//
+// Position of the segment is a position of the arrowhead: GetPointId(true).
+// This position used in heuristic and edges weight calculations.
+class Segment final
+{
+public:
+  Segment() = default;
```

Comment:
Can we change Segment data after initialization? If not, can we declare them as const? Do you really need a default constructor?